src/libs/actions/PolicyConnections.ts

```diff
@@ -0,0 +1,15 @@
+import * as API from '@libs/API';
+import type {OpenPolicyAccountingPageParams} from '@libs/API/parameters';
+import {READ_COMMANDS} from '@libs/API/types';
+
+function openPolicyAccountingPage({policyID}: {policyID: string}) {
```

Comment:
This can be simplified to only pass `policyID` and it doesn't need to accept an object.

Reply:
I changed the parameter in https://github.com/Expensify/App/pull/39132#:~:text=fix%3A%20conditional%20statement%20%26%20change%20the%20parameter%20from%20object%20to%20sing%E2%80%A6